k8s.gcr.io/manifests/legacy-backfill/promoter-manifest.yaml

```diff
@@ -0,0 +1,14 @@
+#
+# WARNING: THIS FILE IS FROZEN. MODIFICATIONS TO THIS FILE ARE FORBIDDEN.
```

Comment:
We should go a step further and add a Prow blockade once this merges.

Reply:
TIL about the blockade plugin. In the same spirit, we should also have a Prow blockade for *all* existing `promoter-manifest.yaml` files, as they should only be touched 1x.

So, (1) blockade the `images.yaml` file in this PR, and (2) blockade all changes to all `promoter-manifest.yaml` files. I can do both in a follow-up PR.

Reply:
Opened PR against test-infra for this here: https://github.com/kubernetes/test-infra/pull/16594